docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,114 @@
+# Google Cloud Platform
+
+## Requirements
+
+This quick start assumes you have administrator access to your Google Cloud Platform instance. 
+
+## Setting Up your VM  
+
+If this is the first time you’re creating a VM on this project, you will need to enable the `Compute Engine API`. Click `enable` and wait for the service to finish connecting.
+
+This will pull up the `Create an Instance` page - name the instance something relevant to this project, following your naming convention of choice. In addition, select a region and zone that is relevant / close to your physical location. For this instance, we will use the base suggested image. 
+
+<img src="../images/quickstart/google-cloud-platform/gcp1.png">
+
+Under `Identity and API Access`, and click `Allow full access to all Cloud APIs`. Scroll down to `Firewall` and click `Allow HTTPS traffic` and `Allow HTTP traffic`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp2.png">
+
+Scroll down to the bottom and click `Create` to create and deploy the VM.
+
+Congrats you’ve created your VM instance!
+
+## SSH-ing into the VM
+
+On the Compute Engine Dashboard, click on the VM for this project. Under `Details`, click `SSH` and select `Open in browser window`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp3.png">
+
+This will give you a terminal to maneuver, manipulate the VM, and install Coder.
+
+## Install Coder
+
+In the terminal, run the following command 
+
+```sh
+curl -fsSL https://coder.com/install.sh | sh  
+```
+
+## Run Coder
+
+For this tutorial, we will run Coder as a System service. You can run Coder in [a multitude of different ways](https://coder.com/docs/coder-oss/latest/install).
+
+First, edit the `coder.env` file to enable `CODER_TUNNEL` by setting the value to true with the following command:
+
+``` sh
+sudo vim /etc/coder.d/coder.env
+``` 
+
+<img src="../images/quickstart/google-cloud-platform/gcp4.png">
+
+Exit vim and run the following command to start Coder as a system service:
+
+```sh
+sudo systemctl enable --now coder
+``` 
+
+The following command will get you information about the Coder service that is running and is also where the access URL for this Coder instance is written. 
+
+```sh
+journalctl -u coder.service -b 
+``` 
+
+This will return a series of logs from launching Coder, however, embedded in the launch is the URL for accessing Coder. 
+
+<img src="../images/quickstart/google-cloud-platform/gcp5.png">
+
+In this instance, Coder can be accessed at the URL  `https://fcca2f3bfc9d2e3bf1b9feb50e723448.pit-1.try.coder.app`. 
+
+Copy the URL and run the following command to create the workspace admin:
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts and be sure to save use email and password. These are your admin username and password. 
+
+You can now access Coder on your local machine by navigating to the `***.try.coder.app` URL and logging in with the username and password. 
```

Comment:
```suggestion
You can now access Coder from your local machine by navigating to the `***.try.coder.app` URL and logging in with the username and password. 
```
This sentence is too similar to the above one, and repeats `username and password`.